practice/bio/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hekate V. Nichols</title>
+     <link rel="stylesheet" href="site.css">
+  </head>
+  <body>
+    <header>
+            <h1>Mx. Hekate V. Nichols</h1>
+    </header>
+      <img src="https://avatars2.githubusercontent.com/u/17284573?v=3&s=460">
+    <blockquote>
+      "What you see depends on where you stand."
+    </blockquote>
+    <section>
+      <article>
+        An ambitious production manager for a theatre company that she built
+        with local artists, who is looking to break into the programming world.
+      </article>
+    </section>
+    <section>
+      <h2>Where has she lived?</h2>
+      <ul>
+        <li>Honolulu, Hawaii</li>
+        <li>Spokane, Washington</li>
+        <li>Moscow, Idaho</li>
+        <li>Getxo, Spain</li>
+        <li>Madison, New Jersey</li>
+        <li>Porland, Oregon</li>
+      </ul>
+    </section>
+    <nav>
+      <footer> Source:
+        <a href="https://www.facebook.com/profile.php?id=40309031">
+          Mx.Hekate Vivika Nichols</a>
+        </footer>
+      </nav>
```

Comment:
There's no requirement that links must appear in a `nav` section, or that a `nav` must appear on each page. I think this link is best described by living just plain in a `footer`.